### PR TITLE
fix: Showing Draft Button for Step-3 & 4

### LINKED ISSUE #3265: Wizard: Show Draft button not appearing in Step 3 & 4
Parent Issue: #3186 

- As far as `Location Field` is considered, We need discussion for it, As an event can also be Online or Have `Location` To-Be-Decided on the basis of the response.

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #3265 

#### Short description of what this resolves:

- Showing the `Save Draft` button if the event is not yet published in every wizard step.

#### Screenshot : 

image

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 Please Review

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 Please Review

File: app/templates/components/forms/wizard/attendee-step.hbs [hunk 1]
```diff
@@ -104,10 +104,12 @@
       {{t 'Forward'}}
       <i class="right chevron icon"></i>
     </button>
-    <button class="blue ui three field right labeled icon button {{if isLoading 'disabled'}}" type="button" {{action 'saveDraft'}}>
-      {{t 'Save draft'}}
-      <i class="save icon"></i>
-    </button>
+    {{#if (not-eq data.event.state 'published')}}
```
Comment (uds5501) [inline app/templates/components/forms/wizard/attendee-step.hbs]: The button should be visible regardless the event state.

File: app/templates/components/forms/wizard/sessions-speakers-step.hbs [hunk 2]
```diff
@@ -320,7 +320,7 @@
       {{t 'Previous'}}
       <i class="left chevron icon"></i>
     </button>
-    {{#if (eq data.event.state 'unpublished')}}
+    {{#if (not-eq data.event.state 'published')}}
```
Comment (uds5501) [inline app/templates/components/forms/wizard/sessions-speakers-step.hbs]: Same

File: app/templates/components/forms/wizard/attendee-step.hbs (hunk 1 above)
Comment (kushthedude) [reply app/templates/components/forms/wizard/attendee-step.hbs]: I don't think , We need this for an already published event and it was implemented the same way in `Basic Detail Form`.

File: app/templates/components/forms/wizard/sessions-speakers-step.hbs (hunk 2 above)
Comment (kushthedude) [reply app/templates/components/forms/wizard/sessions-speakers-step.hbs]: https://github.com/fossasia/open-event-frontend/pull/3266#discussion_r303194213

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 Please Review

**Comment by kushthedude** [comment]
> @kushthedude I think that creating sub-issues for such bugs and then working on such small PRs should be avoided. The issue #3186 clearly stated that all the steps must have `Draft` and `Publish` buttons and if any info is missing, there should be a prompt `You need to fill in the ... [info about missing information here] before you can publish your event`
> You should work on the bug as a whole IMO

https://github.com/fossasia/open-event-frontend/issues/3186#issuecomment-509335622

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 Please Review

**Comment by kushthedude** [comment]
@mrsaicharan1 @shreyanshdwivedi please review

**Comment by kushthedude** [comment]
@shreyanshdwivedi @uds5501 @mrsaicharan1 Please REview

**Comment by kushthedude** [comment]
> [#3266 (review)](https://github.com/fossasia/open-event-frontend/pull/3266#pullrequestreview-268299571)
> 
> If event has not been published, there should be a publish event button.

This check is only for published events while editing them

**Comment by kushthedude** [comment]
@CosmicCoder96 Please Review

**Pull request merged**